app/lib/attachment_batch.rb

```diff
@@ -78,9 +78,17 @@ def remove_files
             logger.debug { "Deleting #{attachment.path(style)}" }
 
             begin
+              retries ||= 0
```

Comment:
There is no need to look at `retries` and see if it needs initializing on every retry. It is probably better to initialize this once before `begin` and be done with it.

```ruby
retries = 0
begin
  # ...
```

Reply:
Good idea. Changed, thanks.